prereise/gather/demanddata/transportation_electrification/data_helper.py

```diff
@@ -63,6 +63,37 @@ def load_data(census_region: int, filepath: str = "nhts_census_updated.mat"):
     return census_data
 
 
+def load_hdv_data(veh_type, filepath: str = "fdata_v10st.mat"):
+    """Load the data at fdata_v10st.mat.
+
+    :param str filepath: the path to the matfile.
+    :return: (*pandas.DataFrame*) -- the data loaded from fdata_v10st.mat, with column
+        names added.
+    """
+
+    hdv_data = loadmat(filepath)
+    raw_data = hdv_data[f"{veh_type}_data"]
+
+    return pd.DataFrame(raw_data, columns=const.hdv_data_column_names)
+
+
+def load_scaling_factor(hdv_year, hdv_cat, filepath = "hdv_daily.mat"):
+    """Load the scaling factor from data of daily VMT values based on
+    the input year and type of HDV vehicle.
+
+    :param int hdv_year: index of input year from this list,
+        [2005, 2012, 2030, 2035, 2040, 2050].
+    :param int hdv_cat: the HDV category, 1-Light-Heavy Duty, 2-Medium-Heavy Duty, 
+        3-Heavy-Heavy Duty
+    :param str filepath: the path to the matfile.
+    :return: (*int/float*) -- the daily VMT of the HDV vehicle.
+    """
+    hdv_daily_mat = loadmat(filepath)
+    hdv_daily_vmt = hdv_daily_mat["hdv_daily_vmt"]
+
+    return hdv_daily_vmt[hdv_year-1, hdv_cat-1]
```

Comment:
Probably want to have check that input year is in the list mentioned in the doc strings